### PR TITLE
fixes suggested by brew audit --strict 

### PR DESCRIPTION
**Pull request by mohakshah**
Made several modifications to formulae as suggested by `brew audit --strict`.

### CODE REVIEW

**Comment by jacknagel** [comment]
Thanks for the pull request. Right now we're not taking style-only or hash-only changes, preferring to fix them when there are other, more substantial changes that need to be made (such as a version update).

The commits that add tests are good, though, please rebase this to contain only those commits.

File: Library/Formula/aescrypt-packetizer.rb [hunk 1]
```diff
@@ -62,10 +60,17 @@ def caveats; <<-EOS.undent
...
+      name = "aescrypt"
+    else
+      name = "paescrypt"
+    end
+
+    system bin/name, "-e", "-p", "fire", path
     assert File.exist?("#{path}.aes")
 
-    system bin/"paescrypt", "-d", "-p", "fire", "#{path}.aes"
+    system bin/name, "-d", "-p", "fire", "#{path}.aes"
```
Comment (DomT4) [inline Library/Formula/aescrypt-packetizer.rb]: We don't normally bother trying to test for the situation when the user has chosen to override a Homebrew default, which is why I didn't bother doing so here before. Jack commented earlier, he'll know one way or another.

**Comment by mohakshah** [comment]
@jacknagel O.K. I just wanted to get more familiar with the process, having applied for gsoc. Anyway, should I keep the updated test block in aescrypt-packetizer.rb?

**Comment by mohakshah** [comment]
Also, I have a few questions regarding test blocks for other formulae. What is a good place to ask those?

**Comment by mohakshah** [comment]
_shntool_'s url needs updating. Should I send the full commit along with style and hash updates or one with just the updated url?

**Comment by mikemcquaid** [comment]
> @jacknagel O.K. I just wanted to get more familiar with the process, having applied for gsoc. Anyway, should I keep the updated test block in aescrypt-packetizer.rb?

Just leave it as it was, thanks.

> Also, I have a few questions regarding test blocks for other formulae. What is a good place to ask those?

Just ask them here.

> shntool's url needs updating. Should I send the full commit along with style and hash updates or one with just the updated url?

With style and hash updates, thanks!

**Comment by mohakshah** [comment]
Is it o.k. to hardcode a test fixture's hash inside the test block and then compare it during the test. Or should I assume it may change in the future?

**Comment by xu-cheng** [comment]
> Is it o.k. to hardcode a test fixture's hash inside the test block and then compare it during the test. Or should I assume it may change in the future?

You can the hash directly:
```ruby
file = test_fixtures("test.png")
file.sha256 # or file.sha1
```

**Comment by mohakshah** [comment]
> You can get the hash directly:
> 
> file = test_fixtures("test.png")
> file.sha256 # or file.sha1

Sorry for the confusion. I see my question was a little ambiguous.
I was writing the test for a tool which can compute the hash of the inner PCM data of a wav file. This hash won't be equal to the hash of the whole file. If the test fixtures are unlikely to change/removed in the foreseeable future, I believe it would be o.k. to simply hardcode the hash of the PCM data into the test block. So, is it likely that fixtures will be changed in the future?

**Comment by xu-cheng** [comment]
> is it likely that fixtures will be changed in the future?

It's unlikely but not guaranteed. It's possible another formula requires a specific fixture file which will result to fixture being altered. 

In your case, I suggest that you only to compute the hash in the test case. Matching the hash with an expected value is not mandatory.

**Pull request closed without merging**